MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Layout in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the Layout support in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Layout in .NET MAUI DataForm (SfDataForm)
+
+## Overview
+
+The data form supports linear and grid layouts and manages layout of label, editor, and validation label.
```

Comment:
The data form supports linear and grid layouts and manages the layout of the label, editor, and validation label.

Reply:
not addressed yet

Reply:
Content changed